pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -319,6 +319,10 @@ export function mounting_dialog(client, block, mode, forced_options) {
                         opts.push("_netdev");
                     if (forced_options)
                         opts = opts.concat(forced_options);
+                    if (subvol) {
+                        opts.push("subvol=" + subvol.pathname);
+                        opts.push("subvolid=" + subvol.id);
```

Comment:
I think specifying both subvol and subvolid is not needed and seems to be asking for trouble if they ever do not match anymore.

Reply:
`subvol` and `subvolid` should be hidden in the UI and not appear in "Custom options", for example.

Reply:
Hmm when do they show up actually?

Reply:
In the "update" version of mounting_dialog. This is caused by not passing the right forced_options and subvol all the way from MountPoint to mounting-dialog.